### ISSUE TITLE
TAGS / UTAGS seem funky ( + future BND support)

### ISSUE THREAD

**Issue by edawson** [zeeev/mergeSVcallers#6]
I'm merging calls from three callers (delly, wham, dRanger), but I don't get what I would expect in the TAGS/UTAGS fields of my vcf. Some tags say "DELLY", which seems right, but others have the sample name. None say "WHAM". I've provided tags on the command line. I guess it's possible that the wham calls are a strict subset of delly's but I would find that a bit surprising.

dRanger only outputs breakends, which it looks like mergeSVcallers still skips. @zeeev If I add some logic to vcflib to combine matched pairs of BNDs into single calls would you be willing to merge in some changes? Do you have any suggestions or requests to make the conversion / merging easy to work with?

This has me stuck in postprocessing purgatory on a project with an ever-approaching deadline.

**Comment by zeeev** [comment]
@edawson Yes, something is off, not sure why.  If you send me an example (a few files and command lines), to my personal email (zev dot kronenberg at gmail.com), I'll start trouble shooting in 20 min when I get into work.
